### PR TITLE
I18n

### PR DESCRIPTION
**Pull request by storresy**
**Summary**
- Added I18n gem
- Internationalized new rent email
- Added locale column to users

**Trello Card**

https://trello.com/c/gxNXsFZK

**Screenshots**

screen shot 2018-12-13 at 18 22 47

### CODE REVIEW

File: app/views/rent_mailer/new_rent_email.slim [hunk 1]
```diff
@@ -1,12 +1,15 @@
 
 h1 
-  | You have registered a new rent for the book 
+  = t('rent_mailer.new_rent.body.title') 
+  '
```
Comment (Gmarn) [inline app/views/rent_mailer/new_rent_email.slim]: Can I have an explication on this?

Comment (storresy) [reply app/views/rent_mailer/new_rent_email.slim]: Yes, that's to add a space between the translated text and the book name

Comment (storresy) [reply app/views/rent_mailer/new_rent_email.slim]: I tried '<=' and '>='  to add the space at the end of those lines but that didn't work so I decided to add the space in the translation text.

Comment (Gmarn) [reply app/views/rent_mailer/new_rent_email.slim]: Perfect 👍

File: app/models/user.rb [hunk 2]
```diff
@@ -1,9 +1,12 @@
...
+
   devise :database_authenticatable, :registerable, :validatable
 
   include DeviseTokenAuth::Concerns::User
 
   has_many :rents, dependent: :nullify
 
-  validates :first_name, :last_name, presence: true
+  validates :first_name, :last_name, :locale, presence: true
+  validates :locale, inclusion: LOCALES
```
Comment (Gmarn) [inline app/models/user.rb:11]: 👏

**Review by Gmarn — APPROVED** [review]

**Pull request merged**